app/helpers/migration_helper.rb

```diff
@@ -0,0 +1,32 @@
+module MigrationHelper
+  def migration_result_attributes(result, key)
+    result.attribute_names.filter { |attribute_name| attribute_name.include?(key) }
+  end
+
+  def migration_result_summary_row(result, key, attribute)
+    value = result.send(attribute)
+    total = result.send("#{key}_count")
+    percentage = number_to_percentage((value.to_f / total) * 100, precision: 0)
+    tag_color = migration_result_percentage_color(key, attribute, value)
+
+    content_tag(:div, class: "govuk-summary-list__row") do
+      content_tag(:dt, attribute.gsub(key.pluralize, "").humanize, class: "govuk-summary-list__key govuk-!-width-one-half") +
+        content_tag(:dd, number_with_delimiter(value), class: "govuk-summary-list__value") +
+        content_tag(:dd, class: "govuk-summary-list__actions") do
+          content_tag(:strong, percentage, class: "govuk-tag govuk-tag--#{tag_color}") if tag_color
```

Comment:
There's [tag support in govuk-components](https://govuk-components.netlify.app/components/tag/#coloured-tags) too.

```ruby
govuk_tag("some text", colour: tag_colour)
```